### PR TITLE
Tweak breakpoints for hot comments 

### PR DESCRIPTION
**Pull request by i-like-robots**
XL
screen shot 2015-08-07 at 11 46 21

L
screen shot 2015-08-07 at 11 46 26

M
screen shot 2015-08-07 at 11 46 35

### CODE REVIEW

File: client/components/stream/_hot-topics.scss [hunk 1]
```diff
@@ -75,16 +75,19 @@
...
 	padding: 10px;
 	color: white;
-	background-color: oColorsGetPaletteColor('red-1');
-	background-color: rgba(oColorsGetPaletteColor('red-1'), 0.8);
 
 	// IE8-9 don't display the image
+	background-color: oColorsGetPaletteColor('red-1');
 	position: static\0;
+
+	background-color: rgba(0, 0, 0, 0.35);
```
Comment (ironsidevsquincy) [inline client/components/stream/_hot-topics.scss]: can we put a jshint warning suppression in here, to stop it complaining about the duplicate property

**Comment by ironsidevsquincy** [comment]
:+1:

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-07/11:03-240/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-07/11:03-240/successes/index.html)

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-07/11:55-245/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-07/11:55-245/successes/index.html)

**Pull request merged**